pnpm-workspace.yaml

```diff
@@ -8,4 +8,5 @@ packages:
   - "apps/wing-console/tools/*"
   - "examples/tests/*"
   - "examples/*"
+  - "scripts/compatibility-matrix-automation"
```

Comment:
Could you move this to the `tools` folder instead? It's not really a script at this point